packages/clay-css/src/scss/atlas/variables/_tables.scss

```diff
@@ -134,6 +122,41 @@ $table-link: map-deep-merge(
 	$table-link
 );
 
+// .table-focus
+
+$c-table-focus: () !default;
+$c-table-focus: map-deep-merge(
+	(
+		td: (
+			before: (
+				box-shadow: (
+					inset 0 0.125rem 0 0 $primary-l0,
```

Comment:
Hey @pat270 this looks annoying to handle, but good job. One question though... do we not have a global variable for focus color that we could use instead of directly using $primary-l0?

Reply:
@ethib137 We have two variables for the focus outline. 

```scss
$component-focus-box-shadow: #{0 0 0 0.125rem $white, 0 0 0 0.25rem $primary-l0} !default;
$component-focus-inset-box-shadow: #{inset 0 0 0 0.125rem $primary-l0,
	inset 0 0 0 0.25rem $white} !default;
```

There are two colors, inner ($white) and outer ($primary-l0). For me, I have to reference these two variables not for the color but for the length settings. I think a global variable for focus color would be useful if we didn't want it to change when someone modifies global colors like `$primary`, `$secondary`, etc.

Reply:
Ahh that makes sense... thanks for explaining the details. Looks good as is.